eu.binjr.binjr.yml

```diff
@@ -0,0 +1,51 @@
+id: eu.binjr.binjr
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk25
+command: binjr
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.gtk.vfs.*
```

Comment:
what is this for?

Reply:
```
--device=dri
--share=network
--filesystem=host
```
Application uses HW accelerated rendering (though OpenGL), needs to access the network and full file system has the underlying framework does not support XDG portals.

```
--talk-name=org.gtk.vfs.*
```
The (flatpak.org documentation)[https://docs.flatpak.org/en/latest/sandbox-permissions.html] states the following, which had me believe this is required to access USB and network shares:
> Typical GNOME and GTK applications should use:
> --talk-name=org.gtk.vfs.*
> --filesystem=xdg-run/gvfsd

While the app is definitely not a "typical GNOME application" it does use GTK. But to be fair, I am unsure wether or not this is needed.

Reply:
I tested without `--talk-name=org.gtk.vfs.*` and it doesn't seem to be needed, so I removed it.